.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -29,15 +29,15 @@ body:
     id: files-version
     attributes:
       label: Files Version
-      description: Which version of Files are you using?
-      placeholder: v1.5
+      description: Which version of Files are you using? Go to the About page in Files' settings, and press "Copy"
```

Comment:
`(Version number can be found by going to Files' Settings > About)`

Reply:
Sounds good!